website/docs/docs/dbt-cloud/setting-up-dbt-cloud/configure-a-repository/managed-repository.md

```diff
@@ -1,11 +1,12 @@
 ---
 title: "Using a dbt Cloud managed repository"
 id: "cloud-using-a-managed-repository"
+sidebar_label: "Managed Repository"
 ---
 
 If you do not already have a git repository for your dbt project, you can let dbt Cloud manage a repository for you.
 
-To create a managed repository, choose the "Managed" option when creating a repository in dbt Cloud. Next, supply the name for the repository -- something like "analytics" or "dbt-models" is a good choice.
+To create a managed repository, choose the "Managed" option when configuring a repository in dbt Cloud. Next, supply the name for the repository -- something like "analytics" or "dbt-models" is a good choice.
```

Comment:
What about "To create a managed repository, choose the "Managed" option when configuring a repository in dbt Cloud during initial set up. Next, supply the name for the repository -- something like "analytics" or "dbt-models" is a good choice."

I _think_ the only way to choose managed repo is when going through the initial set up but could be wrong!